doc/api/fs.md

```diff
@@ -433,6 +433,14 @@ In general, check for the accessibility of a file only if the file won’t be
 used directly, for example when its accessibility is a signal from another
 process.
 
+**NOTE: Usage prior to v6.3.x**
+
+Although this api was added in `v0.11.15` but prior to `v6.3.x`, `fs.constants`
+was `undefined` and all those constants were present directly on `fs`. Thus if
+you are using node version `< v6.3.x` use `fs.R_OK`, etc.
+
+Or you can do something like `(fs.constatns || fs).R_OK` to work with all versions.
```

Comment:
Thank you for the contribution!

A nit: could you please wrap the line <= 80 characters according to [style guide](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md)?

Also, see [commit guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-guidelines): commit title should be <= 50 and use inperative verb (`add`) after the subsystem tag.

All this can be fixed by a collaborator who will land the PR though)